gallery/visualtests/nxTransferListHalf.js

```diff
@@ -64,8 +64,8 @@ describe('NxTransferListHalf', function() {
   });
 
   it('puts a dark border on hovered items', hoverTest(simpleListSelector, secondItemSelector));
-  it('puts a blue border and glow on focused items', focusTest(simpleListSelector, secondItemSelector));
-  it('puts a blue border and blue glow on focused+hovered items',
+  it('puts a blue border on focused items', focusTest(simpleListSelector, secondItemSelector));
+  it('puts a blue border on focused+hovered items',
       focusAndHoverTest(simpleListSelector, secondItemSelector));
   it('puts a grey background on clicked items', clickTest(simpleListSelector, secondItemSelector));
```

Comment:
```suggestion
  it('puts a light grey background on clicked items', clickTest(simpleListSelector, secondItemSelector));
```
(if the other test name suggestions are used - to distinguish between active & hover backgrounds)